worlds/ladx/Options.py

```diff
@@ -303,15 +303,12 @@ class Bowwow(Choice):
 
 class Overworld(Choice, LADXROption):

```

Comment:
Mostly out of curiosity, but why is this called "Overworld" and not something like a Toggle for "OpenMabe"?

Reply:
Future proofing in case I pull some other upstream overworld options (which would be incompatible with open mabe)